src/input.rs

```diff
@@ -16,9 +16,16 @@ pub fn keyval_to_input_string(in_str: &str, in_state: gdk::ModifierType) -> Stri
         debug!("keyval -> {}", in_str);
     }
 
+    // Use the Command key on macOS as Meta/Alt
+    let cmd_as_meta = env::var("NVIM_GTK_CMD_AS_META")
```

Comment:
This feels a bit more painful to me, this module doesn't currently have any state of its own right now, and refactoring all of these functions to take a third parameter for the sake of this doesn't feel very good. Definitely possible though if you're certain, and in that case I think it might as well be a command line argument, since this really shouldn't be changing.

That said, I'm not aware of the performance implications of doing an env lookup on every keypress, I tried just doing a search but I couldn't find anything claiming one way or another, aside from a C++ StackOverflow answer that said they're not really worth caching.

Reply:
I mean - caching this would be very easy (Just use `lazy_static!`) so we might as well. If it's a command line option vs an environment variable though I guess we should go with an env variable. But afaict it shouldn't be difficult to just write a helper function to modify the `ModifierType` that we pass to keyval_to_input_string(). As well, we don't need to have the state for this in input.rs - we could just store it in `shell::State` or `shell::UiState`, etc. and just read it from these functions